articles/tokens/access_token-vs-id_token.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Auth0 access_token vs id_token
+---
+
+# Auth0 access_token vs id_token
+
+## When should you use the `access_token` and the `id_token`?
+
+One of the common questions that gets asked is when you should use the `access_token` and when you should use the `id_token`? 
+
+The answer is as follows:
+
+* If you need to pass a user's credentials to your own API in order to authenticate the user against the API, then you should use the `access_token`
+* When you want to get information about a user from the [/tokeninfo endpoint](/auth-api#post--tokeninfo), then you should use the `id_token`
+
+The situation above is the ideally correct answer, but in reality things are not that simple. Due to historical reasons, `access_token` which is issued by Auth0 is not always a JWT. It used to be a proprietary format which was used to call some of the methods on the [Authentication API](/auth-api).
+
+The JWT format for the `access_token` was only introduced with the new [OAuth2 Authentication Flow](https://auth0.com/docs/api-auth), and you will therefore only be issued a JWT when the user authenticates using the new flow.
+
+If you are still using the older authentication flow, then you should be passing the `id_token` to your own APIs to authentication the user. **Please note** however that passing the `id_token` is strongly discouraged, and you should therefore only use it in this scenarion. If you are using the new OAuth2 authentication flow, then you should be passing the `access_token`.
+
```

Comment:
I'm not sure how to clarify this, but I would omit any discussions about what format the access_token is in.  JWT or something else, it really doesn't matter and doesn't effect the advice around when to use an access_token.

From the client perspective, the access_token is entirely an opaque string of bytes.  It's job is to just send that access token to the resource server for authentication.   The resource server will have to be able to decode the token of course.

APIv1 understands opaque tokens encoded in a db.
APIv2 understands JWT tokens.

Clients don't care.  They just make API calls using whatever token was given to them.